tests/director/tests/notifications/tenant_hierarchy_notifications_test.go

```diff
@@ -86,6 +86,10 @@ func TestFormationAssignmentNotificationsTenantHierarchy(stdT *testing.T) {
 
 		subscriptionToken := token.GetClientCredentialsToken(t, ctx, conf.SubscriptionConfig.TokenURL+conf.TokenPath, conf.SubscriptionConfig.ClientID, conf.SubscriptionConfig.ClientSecret, claims.TenantFetcherTenantHierarchyClaimKey)
 		apiPath := fmt.Sprintf("/saas-manager/v1/applications/%s/subscription", conf.SubscriptionProviderAppNameValue)
+
+		subscription.ConfigureTenantTypeAndID(t, httpClient, subscriptionToken, conf.SubscriptionConfig.URL, "customer", "2c4f4a25-ba9a-4dbc-be68-e0beb77a7eb0")
```

Comment:
Maybe a comment as to where this ID is coming from?
`// Customer is Test_DefaultCustomer`?